src/api/routes/address.ts

```diff
@@ -230,5 +230,22 @@ export function createAddressRouter(db: DataStore, chainId: ChainID): RouterWith
     }
   });
 
+  router.getAsync('/:stx_address/nft_events', async (req, res) => {
```

Comment:
Can you update `docs/openapi.yaml` with this new endpoint?

Reply:
yeah, I was waiting for it to be finalized.